lib/matplotlib/axes/_axes.py

```diff
@@ -1381,8 +1381,8 @@ def plot(self, *args, scalex=True, scaley=True, **kwargs):
 
         Call signatures::
 
-            plot([x], y, [fmt], data=None, **kwargs)
-            plot([x], y, [fmt], [x2], y2, [fmt2], ..., **kwargs)
+            plot([x], y, [fmt], /, *, data=None, **kwargs)
```

Comment:
I have no idea what that means either.  I guess using a short form like this might be OK if the usable;e signatures are explained in detail elsewhere in the doctoring.

Reply:
`*` is standard (valid) Py3 syntax to mark that `data` is keyword only, so I'm definitely leaving it.
`/` is proposed syntax for marking that arguments before it are positional-only (cannot be passed as keywords) (https://www.python.org/dev/peps/pep-0457/#syntax-and-semantics).  I'm only putting this in because people have tried (https://github.com/matplotlib/matplotlib/issues/12106) to pass `x`/`y` as keyword, and this doesn't work (and I don't think it should, for reasons similar to those discussed in https://github.com/matplotlib/matplotlib/issues/11526).  We are already using this syntax for documenting (the admittedly less commonly used) `Axes3D.voxels` (https://matplotlib.org/api/_as_gen/mpl_toolkits.mplot3d.axes3d.Axes3D.html#mpl_toolkits.mplot3d.axes3d.Axes3D.voxels).  Similarly to that case, I added a parenthetical note clarifying this in the Parameters section regarding `x` and `y`.

Reply:
It's not because it is standard valid py3 syntax that it should be used in our examples :)